### PR TITLE
add container add/remove changes in stats sub-command

### LINKED ISSUE #185: add container add/remove changes in `stats` sub-command
add container add/remove changes in `stats` sub-command

### PR DESCRIPTION
**Pull request by xlgao-zju**
close #185

Signed-off-by: Xianglin Gao <xlgao@zju.edu.cn>

### CODE REVIEW

File: api/client/events.go [hunk 1]
```diff
@@ -1,117 +1,88 @@
...
-	flFilter := opts.NewListOpts(nil)
-	cmd.Var(&flFilter, []string{"f", "-filter"}, "Filter output based on conditions provided")
-	cmd.Require(flag.Exact, 0)
-
-	cmd.ParseFlags(args, true)
-
-	eventFilterArgs := filters.NewArgs()
+const (
+	addr = "us-west-1.hyper.sh:443"
+)
```
Comment (carmark) [inline api/client/events.go]: we should get this host from `cli.host` and remove prefix, such as `tcp://`.

File: api/client/events.go [hunk 2]
```diff
@@ -1,117 +1,88 @@
...
+func (cli *DockerCli) Events(ctx context.Context) (<-chan events.Message, <-chan error) {
+	messages := make(chan events.Message)
+	errs := make(chan error, 1)
+
+	go func() {
+		// TODO: add filter when we have other type event.
+		var u = url.URL{Scheme: "wss", Host: addr, Path: "/events/ws"}
+
+		// add sign to header
+		req, err := http.NewRequest("GET", u.String(), nil)
```
Comment (carmark) [inline api/client/events.go]: handle error?

File: api/client/events.go [hunk 3]
```diff
@@ -1,117 +1,88 @@
...
+		req.URL = &u
+		req = signutil.Sign4(cli.configFile.CloudConfig[cli.host].AccessKey, cli.configFile.CloudConfig[cli.host].SecretKey, req)
```
Comment (carmark) [inline api/client/events.go]: should we test whether the `cli.host` is in `cli.configFile.CloudConfig`?

File: vendor/src/github.com/hyperhq/websocket-client/go/util.sh [hunk 4]
```diff
@@ -0,0 +1,217 @@
+#!/bin/bash
+
```
Comment (carmark) [inline vendor/src/github.com/hyperhq/websocket-client/go/util.sh]: It seems that this file is unnecessary.

File: api/client/events.go (hunk 1 above)
Comment (xlgao-zju) [inline api/client/events.go]: All fixed, review plz.

**Comment by xlgao-zju** [comment]
@carmark Updated, please review and try it.

**Pull request merged**